### DISCUSSION TITLE
Issues with building for multiple target triplets when crosscompiling in manifest mode

### DISCUSSION THREAD

**Discussion by equeim** [microsoft/vcpkg#29628] [Q&A]
I'm transitioning to use vcpkg for native dependencies in Android project and I'm facing an issue that manifest mode can only install one target triplet at a time. This means that when Gradle executes CMake for different Android ABIs I need to set up different VCPKG_INSTALLED_DIR for each ABI/triplet, otherwise vcpkg will keep deleting installed packages for other triplets.

The problem with this is that I also have host dependencies. If I have four separate VCPKG_INSTALLED_DIRs then these host dependencies will be duplicated four times which will increase used disk space. Is is possible to set up vcpkg in such a way that host dependencies will be installed in a single directory, while target dependencies will be separated (or tell vcpkg to use multuiple target triplets and install everything in one directory)?

**Comment by autoantwort** [comment]
No that is currently not possible
